catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -355,6 +360,7 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
+                            gson,
```

Comment:
📘 I don't think we're exporting this as a bundle in 2.13.x. Could be wrong though

Reply:
2.8.5 has a manifest. Shouldn't have to embed it.

Reply:
@peterhuffer Has a manifest? What's that mean/do?

Reply:
When I didn't have it embedded, Karaf couldn't find the bundle. Would I have to place it in the feature file or do something extra like that?

Reply:
You will probably have to add this line to your features.xml file if it's not already being pulled in transitively:
```
        <bundle>mvn:com.google.code.gson/gson/${gson.version}</bundle>
```